src/Jonathan_Requirements/Jonathan_Requirements.c

```diff
@@ -0,0 +1,51 @@
+// Requirements.cpp
+// Created by Jonathan Morelock 3/12/2021
+#include <math.h>
+#include "Jonathan_Requirements.h"
+
+int set_draw_distance(int *render_radius, float speed, Precipitation * P)
+{
+    if (speed < 5 || P->type == RAIN)
+    {
+        *render_radius = 5;
+    }
+    if (speed >= 5 && speed <= 10 || P->type == SNOW)
```

Comment:
`else if if (speed >= 5 && speed <= 10 || P->type == SNOW)`

Reply:
I initially had them all as regular if statements then changed them to else if. Must have missed one, thanks!